app/src/main/java/com/example/mycalculator/mvp/view/CalculatorView.java

```diff
@@ -17,5 +19,19 @@ public CalculatorView(MainActivity mainActivity, ActivityMainBinding binding) {
     public void drawNumber(String number) {
         binding.screenTextExample.setText(number);
     }
+    @Override
+    public void showErrorDivision(String errorMessage) {
+        binding.screenTextExample.setText(getContext().getString(R.string.error_message_in_division));
+    }
+    @Override
+    public void showErrorInvalidOperation(String errorMessage) {
+        binding.screenTextExample.setText(getContext().getString(R.string.error_message_when_invalid_format));
+    }
+    @Override
+    public void showErrorMessage(String errorMessage) {
```

Comment:
you are not using the parameter variable, remove it pls

Reply:
Done

Reply:
this still here